Makefile

```diff
@@ -0,0 +1,31 @@
+
+IMG_TAG ?= latest
+IMG ?= quay.io/vshn/kube-token-refresher:$(IMG_TAG)
+
+all: fmt vet build
```

Comment:
Would be cool to add the [help target](https://github.com/projectsyn/component-keycloak/blob/master/Makefile#L18) :)